### DISCUSSION TITLE
Keyball61 パッドが剥がれてしまい、LEDがつかなくなってしまいました。

### DISCUSSION THREAD

**Discussion by g2112026** [Yowkees/keyball#257] [Q&A] [answered]
LEDをうまく付けられなかったため、付け直そうとLEDを外した時に、LEDの左上と右下のパッドとパターン？が剥がれて切れてしました。
修復することはできるのでしょうか？
ご教示いただけますと幸いです。
![IMG_6485](https://github.com/Yowkees/keyball/assets/42173159/2fc8025b-5373-4c0c-aaac-aaf0c47ce7cb)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
この基板はKeyball61のトラックボール有り側で合っていますでしょうか？
下記画像の同色どうしをエナメル線かゴム被覆導線で接続してください。これでLED復活できると思います。
お手数ですがご対応お願いいたします。
![KB61_LEDfix](https://github.com/Yowkees/keyball/assets/84209772/741bf9a0-dba0-4005-8820-2420dc52f441)

**Reply by g2112026** [reply]
ご回答ありがとうございます。
Keyball61のトラックボール有り側の基盤です。
教えていただいた方法で修正したところ、LEDが復活しました！
ありがとうございます。引き続き進めたいと思います。
また何かありましたら、ご連絡させていただくかもしれません。
よろしくお願いいたします。
